src/TodoList.js

```diff
@@ -4,44 +4,20 @@ import TodoListItem from './TodoListItem';
 
   
 
-const todoList =[ 
-  { 
-    id:1,
-    title:"Take out the dog", 
-  }, 
 
-  { 
-    id:2, 
-    title:"Feed the dog",
-  }, 
 
-  { 
-    id:3, 
-    title:"Go to the store"
-  }, 
-
-  {
-    id:4, 
-    title:"Pickup dry cleaning"
-  }, 
-
-  { 
-    id:5,
-    title:"Exercise"
-  }
-]
-
-const TodoList = () =>{ 
+const TodoList = ({todoList}) =>{ 
     return( 
       <ul> 
         {todoList.map(function(item){ 
+          // we are mapping through the state here which is an empty array
```

Comment:
We're actually mapping through the props here. It is the state of the `App` component but the `TodoList` component doesn't know that. That's one of the advantages of React is how reusable and extensible the components are.